Bug: font-size: clamp() reported as error incorrectly

### Environment

ESLint version: 9.25.0
@eslint/css version: 0.8.1
Node version: 22.13.1
npm version: 10.9.2
Operating System: Mac OS 15


### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
import js from "@eslint/js";
import globals from "globals";
import reactHooks from "eslint-plugin-react-hooks";
import reactRefresh from "eslint-plugin-react-refresh";
import tseslint from "typescript-eslint";
import css from "@eslint/css";

export default tseslint.config(
  { ignores: ["dist"] },
  {
    extends: [js.configs.recommended, ...tseslint.configs.recommended],
    files: ["**/*.{ts,tsx}"],
    languageOptions: {
      ecmaVersion: 2020,
      globals: globals.browser,
    },
    plugins: {
      "react-hooks": reactHooks,
      "react-refresh": reactRefresh,
    },
    rules: {
      ...reactHooks.configs.recommended.rules,
      "react-refresh/only-export-components": [
        "warn",
        { allowConstantExport: true },
      ],
    },
  },
  {
    files: ["**/*.css"],
    language: "css/css",
    ...css.configs.recommended,
  },
);
```
</details>

```css
  .fluidHeading {
    font-size: clamp(2.1rem, calc(7.2vw - 0.2rem), 2.5rem);
  }
```


### What did you expect to happen?

I expected that this would lint with no errors.

### What actually happened?


  3:16  error  Invalid value 'clamp(2.1rem,calc(7.2vw - 0.2rem),2.5rem)' for property 'font-size'. Expected <absolute-size> | <relative-size> | <length-percentage [0,∞]> | math  css/no-invalid-properties

✖ 1 problem (1 error, 0 warning)

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-du8wqxzy?file=src%2Fstyle.css

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

_No response_

--- a/package.json
+++ b/package.json
@@ -94,7 +94,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@eslint/core": "^0.14.0",
-    "@eslint/css-tree": "^3.3.3",
+    "@eslint/css-tree": "^3.6.1",
     "@eslint/plugin-kit": "^0.3.1"
   },
   "devDependencies": {

--- a/tests/rules/no-invalid-properties.test.js
+++ b/tests/rules/no-invalid-properties.test.js
@@ -36,6 +36,7 @@ ruleTester.run("no-invalid-properties", rule, {
 		":root { --my-color: red; }\na { color: var(--my-color) }",
 		":root { --my-color: red; }\na { color: var(   --my-color   ) }",
 		":root { --my-color: red;\n.foo { color: var(--my-color) }\n}",
+		".fluidHeading {font-size: clamp(2.1rem, calc(7.2vw - 0.2rem), 2.5rem);}",
 		{
 			code: "a { my-custom-color: red; }",
 			languageOptions: {

